Translations/translation_RU.json

```
@@ -67,35 +67,35 @@
                 "Параметры",
                 "питания"
             ],
-            "desc": ""
+            "desc": "Настройки питания (от аккумуляторов, QC, PD и т.п."

```

```suggestion
            "desc": ""
```
Since it is not used, it was purposely removed!